packages/server/src/restaurants/dishes/api/PartnerDishController.ts

```diff
@@ -58,9 +59,8 @@ class PartnerDishController {
   @ApiObjectIdParam()
   @ApiUpdate({ name: 'dish' })
   @ApiAuthorization(Role.Partner)
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   async update(@PartnerId() restaurant: string, @Param('id') dishId: string, @Body() updateDishDto: UpdateDishDto) {
-    return null; // TODO: Hook up UpdateDishHandler, remove eslint-disable comment above
+    return this.updateDishHandler.exec({ restaurantId: restaurant, id: dishId, ...updateDishDto });
```

Comment:
Możesz zmienić nazwy parametrów na `restaurantId` i `id`, żeby potem móc podać w skróconej formie (`this.updateDishHandler.exec({ restaurantId, id, ...updateDishDto })`), ale to tak tylko stylistycznie.